examples/example-ios/iosBleSdkTestApp/iosBleSdkTestApp/PolarBleSdkManager.swift

```diff
@@ -1182,6 +1284,10 @@ fileprivate extension PolarDeviceDataType {
             return "MAG"
         case .hr:
             return "HR"
+        case .temperature:
+          return ""
```

Comment:
Datatype stringValues need to be defined. 
Use "TEMP" for .temperature.
Use "PRE" for .pressure

Reply:
Please also add temperature and pressure cases into method above:
`func dataToString<T>(_ data: T) -> String `